Pat.Subscriber/SubscriptionBuilder.cs

```diff
@@ -56,8 +58,19 @@ private async Task BuildSubscription(string connectionString, string[] messagesT
         {
             var topicName = _config.EffectiveTopicName;
 
-            var client = new SubscriptionClient(connectionString, topicName, _config.SubscriberName);
-         
+            SubscriptionClient client;
+            if (_config.TokenProvider != null)
+            {
+                ServiceBusConnectionStringBuilder builder = new ServiceBusConnectionStringBuilder(connectionString);
+                Console.WriteLine(builder.Endpoint);
```

Comment:
Did we mean to leave the console log in?

Reply:
nope have removed